itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestJdbcDriver2.java

```diff
@@ -2793,6 +2795,124 @@ public void run() {
     verifyFetchedLog(incrementalLogs, expectedLogs);
   }
 
+  private static  int next = 0;
+  private synchronized void advanceDumpDir() {
+    next++;
+    ReplDumpWork.injectNextDumpDirForTest(String.valueOf(next));
+  }
+
+  /**
+   * Test getting query log method in Jdbc for REPL commands
+   * @throws Exception
+   */
+  @Test
+  public void testGetQueryLogForReplCommands() throws Exception {
+    // Prepare
+    String primaryDb = "primaryDb";
+    String replicaDb = "replicaDb";
+    String primaryTblName = "primaryDb.t1";
+    String sql = "select count(*) from " + tableName;
+    Path replDir = new Path(conf.get("test.data.files"));
```

Comment:
Is it hardcoded path? Why don't we drop the directory pointed by hive.repl.rootdir config?

Reply:
its not hard coded path ..its within configured test folder ..easy to maintain

Reply:
So, it is set in WITH clause. OK fine.